README.md

```diff
@@ -44,19 +44,37 @@ To build the operator:
     make build
     ```
 
-## Updating Custom Resource Definitions
-
-The custom resource definitions used by the Verrazzano Operator are auto generated by the Verrazzano project `verrazzano-crd-generator`.
-The steps to include changes made to the custom resource definitions are:
-
-* Make the API changes in the `verrazzano-crd-generator` project
-* Tag a branch of `verrazzano-crd-generator`, for example `0.n`
-* Edit `go.mod` to use the newly tagged API changes.  For example `github.com/verrazzano/verrazzano-crd-generator v1.2.0`
+## Updating Dependencies
+
+The dependencies used by the Verrazzano Operator are managed by go's `go.mod` file.
```

Comment:
by go's `go.mod` file. -> by the `go.mod` file.

Reply:
Fixed.